controllers/serviceinstance_controller_test.go

```diff
@@ -751,13 +711,7 @@ var _ = Describe("ServiceInstance controller", func() {
 
 				It("should not delete the k8s instance and condition is updated with failure", func() {
 					deleteInstance(ctx, serviceInstance, false)
-					Eventually(func() bool {
-						err := k8sClient.Get(ctx, defaultLookupKey, serviceInstance)
-						if apierrors.IsNotFound(err) {
-							return false
-						}
-						return isFailed(serviceInstance)
-					}, timeout, interval).Should(BeTrue())
+					waitForInstanceToBeFailed(ctx, defaultLookupKey)
```

Comment:
same as earlier comment, don't use this method

Reply:
done